vortex-api/src/main/java/mil/army/usace/hec/vortex/VortexGrid.java

```diff
@@ -310,9 +353,9 @@ public boolean equals(Object o) {
         if (Double.compare(that.originX, originX) != 0) return false;
         if (Double.compare(that.originY, originY) != 0) return false;
         if (Double.compare(that.noDataValue, noDataValue) != 0) return false;
-        if (ReferenceUtils.equals(wkt, that.wkt)) return false;
+        if (!ReferenceUtils.equals(wkt, that.wkt)) return false;
         if (!Arrays.equals(data, that.data)) return false;
-        if (!Objects.equals(units, that.units)) return false;
+        if (!units.equalsIgnoreCase(that.units)) return false;
```

Comment:
This is dangerous if for some reason the first `units` in the equality is `null`. An NPE will be thrown and things will fail. I would add `null` checks.